src/components/Footer.css

```diff
@@ -73,16 +54,80 @@ footer a:hover {
 .footer-content p {
   text-align: left;
   width: calc(100% - 10.375rem);
+  margin-bottom: 0;
 }
 
 .languages {
   clear: both;
-  padding-left: 10px;
-  margin-left: 10px;
+  padding-left: 1.25rem;
+  margin-left: 1.25rem;
   border-left: 1px solid #666;
 }
 
 .languages .form-control {
   width: auto;
   padding: 0;
+}
+
+@media all and (min-width: 767px) {
+  footer .icon .footer-icon {
+    max-width: 18px;
+    margin-right: 5px;
+  }
+}
+
+@media all and (max-width: 767px) {
+  footer .icon-container {
+      display: block;
```

Comment:
minor: can you please indent everything to two spaces? (a bit further down as well)

Reply:
fixed! (: